sirenabailie/Fan-Fusion-FE
# Logo and Color Scheme

## Detailed Description

Added a logo and applied a cohesive color scheme throughout the app to enhance branding and visual appeal. The logo appears in the navbar and login page, while the new color scheme standardizes the app's look and feel across all components.

## How to Test


1.  Start the application and navigate to various pages to confirm the logo appears in the navbar and login page.
2. Check that the color scheme is consistent and visually appealing across all pages and components.

## Motivation and Context

This change is required to improve brand identity and create a consistent, professional look throughout the app, enhancing the user experience.

## Types of Changes
